students/jemielity_kamil/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,10 @@
+import math
+
+N = int(input("Number of students: "))
+K = int(input("Amount of apples: "))
+
+amountOfApplesPerStudent = math.floor(K/N)
```

Comment:
Please use underscores instead of camel case for variable name in python
apples_per_student = math.floor(k / n)